game.exs

```diff
@@ -1,7 +1,52 @@
 defmodule GuessingGame do
+  @moduledoc """
+  A simple Elixir-based guessing game for guessing a number between 1 and 100.
+  """
 
-  def hi() do
-    IO.puts("hello world")
+  @ver 0.1
+
+  def runner do
+    start_game
+    answer = generate_answer
+    game_loop(answer)
+  end
+
+  defp game_loop(answer) do
```

Comment:
Nice, using pipes!

Reply:
I wonder if game_loop could be a longer pipe. Get the guess, pipe it to something that checks, pipe it to something that prints a message, pipe it to something that knows what to do next in the game(continue, abort, etc)...